src/middleware/authorizationMiddleware.js

```diff
@@ -1,5 +1,7 @@
 const config = require('../config/tokenConfig.json');
```

Comment:
The names are not consistent

Reply:
Solved https://github.com/Syncora/Syncora-Backend/pull/12/commits/002fbcbc8fc658bba2099f0ba8a8a5569b87b741